packages/block-directory/src/components/downloadable-block-info/index.js

```diff
@@ -16,7 +16,9 @@ function DownloadableBlockInfo( { description, activeInstalls, humanizedUpdated
 					<Icon icon="chart-line"></Icon>{ sprintf( _n( '%d active installation', '%d active installations', activeInstalls ), activeInstalls ) }
 				</div>
 				<div className="block-directory-downloadable-block-info__column">
-					<Icon icon="update"></Icon><span aria-label={ sprintf( __( 'Updated %s' ), humanizedUpdated ) }>{ humanizedUpdated }</span>
+					<span aria-label={ sprintf( __( 'Updated %s' ), humanizedUpdated ) }>
+						<strong>{ __( 'Updated: ', 'gutenberg' ) }</strong> { humanizedUpdated }
```

Comment:
```suggestion
						<strong>{ __( 'Updated: ' ) }</strong> { humanizedUpdated }
```

We don't use the `gutenberg` textdomain here, because it will be merged into core.